docs/internals/contributing/writing-code/accessibility.txt

```diff
@@ -0,0 +1,114 @@
+=============
+Accessibility
+=============
+
+All aspects of Django must be compatible with our accessibility standards,
+and with the specific support targets we aim for.
+
+.. _accessibility-standards:
+
+Accessibility standards
+=======================
+
+Django aims for conformance with `WCAG 2.2`_ at the AA level.
+WCAG is the most well established standard for web accessibility. AA-level
```

Comment:
```suggestion
WCAG is the most established standard for web accessibility. AA-level
```
"most established" is sufficient 👍

Reply:
Thanks! English isn’t my native language so very much appreciate that kind of feedback.